core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci

```diff
@@ -87,4 +87,4 @@ quay.io/kiali/kiali-cypress-tests:v1.57 registry.ci.openshift.org/ci/kiali-cypre
 quay.io/kiali/kiali-cypress-tests:v1.65 registry.ci.openshift.org/ci/kiali-cypress-tests:v1.65
 quay.io/redhat_msi/managed-services-integration-tests:latest registry.ci.openshift.org/ci/managed-services-integration-tests:latest
 quay.io/rh_integration/xpaas-qe:7.11.x-openjdk1.8_local registry.ci.openshift.org/ci/xpaas-qe:7.11.x-openjdk1.8_local
-quay.io/coreos/tectonic-console-builder:v25 registry.ci.openshift.org/ci/tectonic-console-builder:v25
+quay.io/coreos/tectonic-console-builder:v25 registry.ci.openshift.org/ci/tectonic-console-builder:v25
```

Comment:
This change is not required for this PR, please revert back this file

Reply:
Updated.